spec/validators/word_count_validator_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe WordCountValidator do
+  maximum = 10
+
+  before do
+    stub_const('Validatable', Class.new).class_eval do
+      include ActiveModel::Validations
+      attr_accessor :some_words
+      validates :some_words, word_count: { maximum: maximum }
+    end
+  end
+
+  let(:model) {
+    model = Validatable.new
+    model.some_words = some_words_field
+    model
```

Comment:
A way of avoiding 3 lines here

```suggestion
    Validatable.new.tap { |model| model.some_words = some_words_field } 
```